### ISSUE TITLE
Strange scrollbars and white right border in text editor (not in the workspace)

### ISSUE THREAD

**Issue by gusbemacbe** [atom/atom#16142]
### Description

There is a white right border in the text editor. The strange scrollbars are not customised by Atom Material, so they are native scrollbar. 

I use Atom Material and Legacy Tron (film TRON The Legacy syntax) or An Old Hope (Star Wars syntax).

### Steps to Reproduce

1. Fullfil the window
2. Click in the text window and touch the right
3. You will see a white right border.
4. Also notice the scrollbars. Compare the transparent scrollbar to the white scrollbar of project sidebar, because they didn't get customised by Atom Material. 

In the original 1.20 or later, with Atom Material activated, scrollbar were beautiful, blue and non-transparent, similar to blue bottom border of tabs. 

**Expected behavior:** [What you expect to happen]

Fulfil and touch the right of the text editor

**Actual behavior:** [What actually happens]

The white right border appears.

**Reproduces how often:** [What percentage of the time does it reproduce?]
Milliseconds 

### Versions

Atom 1.23 Beta
macOS Sierra 10.13.2

### Additional Information

Here are three screenshots:

Open the first image with black background and notice the white border at right:

![White border](https://user-images.githubusercontent.com/2257024/32592024-0df173b2-c509-11e7-8f2a-514ccb21bec8.png)

With Atom Material and An Old Hope:

![With Atom Material and An Old Hope](https://user-images.githubusercontent.com/2257024/32591909-8cacb316-c508-11e7-819e-0067a2523d18.png)

With Atom Material and Legacy TRON:

![With Atom Material and Legacy TRON](https://user-images.githubusercontent.com/2257024/32591979-d3197be0-c508-11e7-9731-f66217febaac.png)

**Comment by gusbemacbe** [comment]
I can't scroll vertically and horizontally in any Atom UI theme and syntax theme.

**Comment by rsese** [comment]
Hmmm, about the issue with Atom Material as your UI theme, this looks like the same report?

https://github.com/atom-material/atom-material-ui/issues/442

> I can't scroll vertically and horizontally in any Atom UI theme and syntax theme.

Do you see this behavior in safe mode (`atom --safe`)?

**Comment by gusbemacbe** [comment]
@rsese, I updated to 1.23.0-beta1, and only the white border has disappeared.

But one white pixel in the bottom right of the scrollbar didn't disappear if I customised the scrollbar.

![screen shot 2017-11-17 at 00 22 51](https://user-images.githubusercontent.com/2257024/32926281-999ecd5e-cb2d-11e7-9a8f-bbe113077d18.png)

Running `atom-beta --safe`, the white pixel also still appears:

![screen shot 2017-11-17 at 00 25 07](https://user-images.githubusercontent.com/2257024/32926344-e0a68c64-cb2d-11e7-885d-e14b44587b8c.png)

De-customising the scrollbar, in both normal Atom and `atom-beta --safe`, the white pixel disappears, but the scrollbar doesn't get customised by Atom Material.

As for scrolling:

a. After running `atom-beta --safe`:

1. Customising the webkit-scrollbar:
I can scroll horizontally and vertically.

2. De-customising the webkit-scrollbar:
I can't scroll horizontally and vertically.

b. In normal Atom:

1. Customising the webkit-scrollbar:
I can scroll horizontally and vertically.

2. De-customising the webkit-scrollbar:
I can scroll vertically, but I can't scroll horizontall.
